### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by mcrd25**
This projects tests 5 different test subjects and methods
- `captilaize(string)`
	- returns a string with the first letter capitalized
	- test ensures that capitalized string is returned
- `reverseString(string)`
	- returns the reverses string
	- the associated test ensures to see the reversed string is returned
- calculator
	- An object that adds, subtracts, divides and multiplies two numbers and returns result
	- Tests ensure that calculator methods `add`, `subtract`, `divide`, and `multiply` return correct results
- `ceaser()`
	- encrypts passed string by passing key shifts while only shifting letters.
	- Test takes care of following scenarios/edge cases
		- wrapping from z to a ( *example: if cipher key is 5 and letter is say w it should wrap and go to b* )
		- Maintains correct case
		- punctuation and spaces remain as is
- array analysis 
	- analyze function returns an object that has array properties average, min, max, and length
	```js
		const object = analyze([1,8,3,4,2,6]);

		object == {
			average: 4,
			min: 1,
			max: 8,
			length: 6
		};
	```
	- test test that object returns correct values of `average`, `min`, `max` and `length`

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ linters:
     config: './.eslintrc.json'
 files:
   ignore:
-    - 'dist/main.js'
\ No newline at end of file
+    - 'tests/*'
```
Comment (zenott) [inline .stickler.yml]: Adding `"jest": true` to the `env` section of .eslintrc.json would have allowed jest globals, it is not necessary to use the jest plugin.
